_posts/0000-01-02-adrivpp.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![what-is-a-meme](https://user-images.githubusercontent.com/35638002/50104629-d65fdf00-022a-11e9-88cd-b2bab7b6e502.jpg)
*Made it*